modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -70,6 +67,12 @@
  * @param <T>
  *            The type of the wrapped value.
  *
+ * @implSpec <ol>
+ * <li> All bindings and properties in this library support lazy evaluation.
+ * <li> Current implementing classes in JavaFX check for a change using reference
+ * equality (and not object equality, {@code Object#equals(Object)}) of the value.
```

Comment:
Now that they are right next to each other, it would be good to unify the language of these two requirements, making it clear that both apply to all JavaFX classes that implement this interface. Perhaps something like:

1. All bindings and properties in the JavaFX library support lazy...
2. All implementing classes in the JavaFX library check for a change...